From 30211b7336f2ed6048953390340e86f6cca9eb6f Mon Sep 17 00:00:00 2001
From: Dominique Leuenberger <dimstar@opensuse.org>
Date: Thu, 17 May 2018 17:40:37 +0200
Subject: [PATCH] qt: fix build against Qt 5.11

Qt 5.11 is in RC phase now, and some headers need to be explicitly
included in order to build successfully.

Signed-off-by: Jean-Baptiste Kempf <jb@videolan.org>

Downloaded from
https://sources.debian.org/src/vlc/3.0.3-1-3/debian/patches/0003-qt-fix-build-against-Qt-5.11.patch/

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 modules/gui/qt/components/simple_preferences.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules/gui/qt/components/simple_preferences.cpp b/modules/gui/qt/components/simple_preferences.cpp
index 635cc274c0..7a31a6ea6f 100644
--- a/modules/gui/qt/components/simple_preferences.cpp
+++ b/modules/gui/qt/components/simple_preferences.cpp
@@ -36,6 +36,7 @@
 #include <QString>
 #include <QFont>
 #include <QToolButton>
+#include <QButtonGroup>
 #include <QSignalMapper>
 #include <QVBoxLayout>
 #include <QScrollArea>
-- 
2.14.4

